manifests/system-configuration.yaml

```
@@ -5,6 +5,8 @@
 packages:
   # Configuring SSH keys, cloud provider check-in, etc
   - afterburn
+  # Adding this subpackage for RHCOS/RHEL
+  - afterburn-dracut
```

I wouldn't worry with an extra comment and just merge them onto a single line. 

```suggestion
  # Configuring SSH keys, cloud provider check-in, etc
  - afterburn afterburn-dracut
```